docs/cli-arguments.md

```diff
@@ -31,6 +31,7 @@ Usage:
 
 Available Commands:
   completion  Generate completion script for kube-state-metrics.
+  generate    Generate custom resource metrics configuration from go-code markers.
```

Comment:
nit: `Generate custom resource metrics configuration from go-code markers.` -> `Generate custom resource metrics configuration from go-code markers (experimental).` 

After more users use/test this feature, we can remove experimental.

Reply:
Thanks, adjusted.